cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -30,11 +30,11 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/apiserver/pkg/server"
 	"k8s.io/apiserver/pkg/server/healthz"
-	apiserverflag "k8s.io/apiserver/pkg/util/flag"
-	"k8s.io/apiserver/pkg/util/globalflag"
 	"k8s.io/client-go/tools/leaderelection"
 	"k8s.io/client-go/tools/leaderelection/resourcelock"
 	cloudprovider "k8s.io/cloud-provider"
+	apiserverflag "k8s.io/component-base/cli/flag"
+	"k8s.io/component-base/cli/globalflag"
```

Comment:
can we rename 'apiserverflag'?
the same in others place

Reply:
done. it is now `cliflag`. On a side note. I did not name any of the `globalflag` imports as it was never named before and left those imports bare.